src/main/pages/che-7/end-user-guide/assembly_defining-custom-commands-for-che-theia.adoc

```diff
@@ -18,78 +18,15 @@ summary:
 
 :context: defining-custom-commands-for-che-theia
 
-The Che-Theia IDE allows users to define custom commands in a devfile that are then available when working in a workspace.
+When starting a new project, new developers often loose time in finding the good procedure to build, run or debug a project. The result is documentation that are hard to follow and never up-to-date.
 
-The following is an example of the `commands` section of a devfile.
+On boarding new developers is one of the main Che challenges. To help the new developers, lead developers can define devfiles to create on-demand workspaces for specific projects. 
```

Comment:
Again, I suggest leaving the orig. wording. If you believe we should better describe the purpose of devfiles, it should be included in the *What is a devfile* section at https://www.eclipse.org/che/docs/che-7/configuring-a-workspace-using-a-devfile/ (which, incidentally, could use a better title).

```suggestion
The following is an example of the `commands` section of a devfile.
```